src/components/SideNavbar/SideNavbar.style.tsx

```diff
@@ -56,13 +56,44 @@ export const SidebarNavList = styled.ul`
   padding: 0;
 `
 
+export const ButtonGroup = styled.div`
+  width: ${EXPANDED_SIDENAVBAR_WIDTH}px;
+  margin-bottom: ${sizes(10)};
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+`
+
+export const ButtonLink = styled(Link)`
```

Comment:
I'm not sure this will fly. We probably should add support for our regular button to be a link. This way we don't need to style it here, we get all the hover/pressed states and we don't need to update it once the design changes

Reply:
Let's wait till the buttons PR is merged and work this out

Reply:
@drillprop merged now, you should be able to use the new buttons for this